### PR TITLE
[mod_breadcrumbs]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_breadcrumbs
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

**Comment by freshweb** [comment]
I have tested this item :white_check_mark: successfully on d06399844604fe4862f1778e56ba9954b01472f6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10578">issues.joomla.org/joomla-cms/10578</a>.</sub>

File: modules/mod_breadcrumbs/mod_breadcrumbs.xml [hunk 1]
```diff
@@ -25,80 +25,94 @@
...
 					name="layout"
 					type="modulelayout"
 					label="JFIELD_ALT_LAYOUT_LABEL"
-					description="JFIELD_ALT_MODULE_LAYOUT_DESC" />
+					description="JFIELD_ALT_MODULE_LAYOUT_DESC"
+				/>
+
 				<field
 					name="moduleclass_sfx"
 					type="textarea" rows="3"
```
Comment (andrepereiradasilva) [inline modules/mod_breadcrumbs/mod_breadcrumbs.xml]: shouldn't the rows be moved down?

File: modules/mod_breadcrumbs/mod_breadcrumbs.xml [hunk 2]
```diff
@@ -25,80 +25,94 @@
...
+					description="COM_MODULES_FIELD_CACHE_TIME_DESC"
+				/>
+
 				<field
 					name="cachemode"
 					type="hidden"
 					default="itemid">
-					<option
-						value="itemid"></option>
+					<option value="itemid"></option>
```
Comment (andrepereiradasilva) [inline modules/mod_breadcrumbs/mod_breadcrumbs.xml]: shouldn't the > be in a new line?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 4ae890d2ea22e23499b1384a44f7c2a858d00a44<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10578">issues.joomla.org/joomla-cms/10578</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10578">issues.joomla.org/joomla-cms/10578</a>.</sub>

File: modules/mod_breadcrumbs/mod_breadcrumbs.xml [hunk 3]
```diff
@@ -25,80 +25,96 @@
...
 				</field>
+
 				<field
 					name="cache_time"
 					type="text"
-					default="0"
 					label="COM_MODULES_FIELD_CACHE_TIME_LABEL"
-					description="COM_MODULES_FIELD_CACHE_TIME_DESC" />
+					default="0"
+					description="COM_MODULES_FIELD_CACHE_TIME_DESC"
```
Comment (JoomliC) [inline modules/mod_breadcrumbs/mod_breadcrumbs.xml]: Description should be after label ;-)

**Comment by JoomliC** [comment]
I have tested this item :red_circle: unsuccessfully on 4ae890d2ea22e23499b1384a44f7c2a858d00a44<br /><br />One CS error : https://github.com/joomla/joomla-cms/pull/10578/files#diff-348cd6cdc38aadafe465038bb27f07a4R107<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10578">issues.joomla.org/joomla-cms/10578</a>.</sub>

**Comment by wojsmol** [comment]
@JoomliC fox pushed - I wait until github notice this.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 17f5c31769916cd52d441ac3c2e6124d8a2e73ee<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10578">issues.joomla.org/joomla-cms/10578</a>.</sub>

**Pull request closed without merging**